### ISSUE TITLE
[BUG] Unable to download runway 2.6.2 from oni.ca

### ISSUE THREAD

**Issue by wenindoubt** [onicagroup/runway#1462]
### Bug Description

Downloading a specific version (2.6.2) from oni.ca does not exist. 
```
$ curl -L https://oni.ca/runway/2.6.2/osx -o runway
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
$ ls -ltr
.rw-r--r--  0 jeff  6 Jun 09:07 runway
```
Runway is 0 bytes

Downloading latest from oni.ca works.

```
$ curl -L https://oni.ca/runway/latest/osx -o runway
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 51.6M  100 51.6M    0     0  12.5M      0  0:00:04  0:00:04 --:--:-- 20.3M

$ ls -ltr
.rw-r--r--  54M jeff  6 Jun 09:09 runway
$ chmod +x runway
$ ./runway --version
2.6.2
```

### Expected Behavior

I expect to be able to download `runway` version `2.6.2` either using `latest` or `2.6.2` from oni.ca.

However, only `latest` works.

### Steps To Reproduce

1. `curl -L https://oni.ca/runway/2.6.2/osx -o runway`
2. No binary

or

1. `curl -L https://oni.ca/runway/2.6.2/linux -o runway`
2. Binary exists and is version 2.6.2

### Runway version

2.6.2

### Installation Type

direct download (curl, wget, etc)

### OS / Environment

- OS: macOS 12.3.1

**Comment by alvarado-fabian** [comment]
Im seeing the same behavior, unable to download via 2.6.2 link. 

$ curl -L https://oni.ca/runway/2.6.2/linux -o runway
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
$
$

**Comment by jake-skipper** [comment]
Hello @alvarado-fabian ,

We are still investigating this one, looks like there may have been a pipeline issue that prevented the upload. In the meantime, we have just released version 2.6.3 which appears to have been uploaded without issues.

We will leave this ticket open to track the existing 2.6.2 problem.

Thanks!

**Comment by wenindoubt** [comment]
> Hello @alvarado-fabian ,
> 
> We are still investigating this one, looks like there may have been a pipeline issue that prevented the upload. In the meantime, we have just released version 2.6.3 which appears to have been uploaded without issues.
> 
> We will leave this ticket open to track the existing 2.6.2 problem.
> 
> Thanks!

Thanks! I've tested 2.6.3 and it works!
